screens/CreateAccScreen.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {
+  Image,
+  Platform,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+  TouchableHighlight
+} from 'react-native';
+import { WebBrowser } from 'expo';
+import HomeScreen from './HomeScreen';
+import { MonoText } from '../components/StyledText';
+import { InputText } from '../components/InputText';
+
+
+export default class IntroAppScreen extends React.Component {
```

Comment:
Why is this named IntroAppScreen when the file is CreateAccScreen?

Reply:
Changed Screen name in local and pushed on new pull req.